src/fsharp/NameResolution.fsi

```diff
@@ -244,6 +247,9 @@ val internal AddModuleOrNamespaceRefToNameEnv                    : TcGlobals ->
 /// Add a list of modules or namespaces to the name resolution environment
 val internal AddEntitiesContentsToNameEnv : TcGlobals -> ImportMap -> AccessorDomain -> range -> bool -> NameResolutionEnv -> ModuleOrNamespaceRef list -> NameResolutionEnv
 
+/// Add the content of a type to the name resolution environment
+val internal AddTypeContentsToNameEnv : TcGlobals -> ImportMap -> AccessorDomain -> range -> NameResolutionEnv -> TType -> NameResolutionEnv
```

Comment:
Perhaps we should just accept a `TyconRef` instead of a `TType` since we will never allow opening non-named types.

Reply:
Might be better to keep `TType` as is.